src/components/DateTimePicker/DateTimePicker.story.jsx

```diff
@@ -28,9 +28,9 @@ export const defaultRelativeValue = {
 export const defaultAbsoluteValue = {
   timeRangeKind: PICKER_KINDS.ABSOLUTE,
   timeRangeValue: {
-    startDate: '04/01/2020',
+    startDate: '2020-04-01',
     startTime: '12:34',
-    endDate: '04/06/2020',
+    endDate: '2020-04-06',
```

Comment:
@davidicus did you figure out that warning with the date formats, or is this okay to merge?